README.md

```diff
@@ -59,10 +59,11 @@ import { Animated, StyleSheet, Text, View, useWindowDimensions } from 'react-nat
 import { PageIndicator } from 'react-native-page-indicator';
 
 const pages = ['Page 1', 'Page 2', 'Page 3'];
+const rtl = false;
 
 const App = () => {
   const { width, height } = useWindowDimensions();
-  const scrollX = useRef(new Animated.Value(0)).current;
+  const scrollX = useRef(new Animated.Value(rtl ? width * (pages.length - 1) : 0)).current;
```

Comment:
Thank you for the PR, this helped me save a lot of time. I just used `bun patch` to generate a patch for my app.

I also needed to change the default value depending on iOS/Android. For some reason on iOS the slider still works from LTR while the ScrollView works RTL, so I used 0 for iOS and windowWidth * (pageLengt-1) for Android.

Do you know how to make the indicator work RTL on iOS as well?